explore-analyze/query-filter/languages/esql-rest.md

```diff
@@ -64,15 +64,21 @@ POST /_query?format=txt
 The URL parameter takes precedence over the HTTP headers. If neither is specified then the response is returned in the same format as the request.
 ::::
 
+::::{tip}
+In the "Tabular" formats, only the resulting data will be returned. Other response fields will be missing.
```

Comment:
```suggestion
In the tabular formats, only the result data is returned. The other response fields, such as those specific to async processing, are returned as HTTP headers.
```

Reply:
@ivancea alternatively you could have 3 little tables and introduce them with as much detail as you need in prose, just some ideas, don't have super strong opinions here :)